Jetcaster/wear/src/main/java/com/example/jetcaster/ui/podcast/PodcastDetailsScreen.kt

```diff
@@ -74,105 +69,146 @@ import com.google.android.horologist.media.ui.screens.entity.EntityScreen
+    onPlayButtonClick: () -> Unit,
+    onPlayEpisode: (List<PlayerEpisode>) -> Unit,
+    onEpisodeItemClick: (PlayerEpisode) -> Unit,
+    columnState: TransformingLazyColumnState,
+    contentPadding: PaddingValues,
+    modifier: Modifier = Modifier,
+) {
+    TransformingLazyColumn(
+        modifier = modifier,
+        state = columnState,
+        contentPadding = contentPadding,
+    ) {
+        item {
+            ListHeader {
+                Text(
+                    text = title, maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
                 )
             }
         }
+        item {
+            ButtonsContent(
+                episodes = episodeList,
+                onPlayButtonClick = onPlayButtonClick,
+                onPlayEpisode = onPlayEpisode,
+            )
+        }
+        items(episodeList) { episode ->
+            MediaContent(
+                episode = episode,
+                episodeArtworkPlaceholder = painterResource(id = R.drawable.music),
+                onItemClick = onEpisodeItemClick,
+            )
+        }
     }
 }
 
-@OptIn(ExperimentalHorologistApi::class)
 @Composable
-fun ButtonsContent(episodes: List<PlayerEpisode>, onPlayButtonClick: () -> Unit, onPlayEpisode: (List<PlayerEpisode>) -> Unit) {
+fun PodcastDetailScreenLoading(columnState: TransformingLazyColumnState, contentPadding: PaddingValues, modifier: Modifier = Modifier) {
+    TransformingLazyColumn(
+        modifier = modifier,
+        state = columnState,
+        contentPadding = contentPadding,
+    ) {
+        item {
+            ListHeader {
+                Text(text = stringResource(id = R.string.loading))
+            }
+        }
+        item {
+            ButtonsContent(
+                episodes = emptyList(),
+                onPlayButtonClick = {},
+                onPlayEpisode = {},
+                enabled = false,
+            )
+        }
+        items(count = 2) {
+            PlaceholderButton()
+        }
+    }
+}
+@Composable
+fun ButtonsContent(
+    episodes: List<PlayerEpisode>,
+    onPlayButtonClick: () -> Unit,
+    onPlayEpisode: (List<PlayerEpisode>) -> Unit,
+    enabled: Boolean = true,
+    modifier: Modifier = Modifier,
+) {
 
-    Chip(
-        label = stringResource(id = R.string.button_play_content_description),
+    Button(
         onClick = {
             onPlayButtonClick()
             onPlayEpisode(episodes)
         },
-        modifier = Modifier.padding(bottom = 16.dp),
-        icon = Icons.Outlined.PlayArrow.asPaintable(),
-    )
+        enabled = enabled,
+        icon = {
+            Icon(
+                painter = painterResource(id = R.drawable.play),
+                contentDescription = stringResource(id = R.string.button_play_content_description),
+            )
+        },
+        modifier = modifier.fillMaxWidth(),
+    ) {
+        Text(stringResource(id = R.string.button_play_content_description))
+    }
 }
 
 @ExperimentalHorologistApi
```

Comment:
consider moving this code to another file since its not ui code

Reply:
done